File: vortexdbg-api/src/main/java/com/vortexdbg/file/VortexdbgFileFilter.java
package com.vortexdbg.file;

import java.io.File;
import java.io.FileFilter;

public class VortexdbgFileFilter implements FileFilter {

    public static final String UNIDBG_PREFIX = "__ignore.vortexdbg";

    @Override
    public boolean accept(File pathname) {
        return !pathname.getName().startsWith(UNIDBG_PREFIX);
    }

}

File: vortexdbg-api/src/main/kotlin/com/vortexdbg/file/VortexdbgFileFilter.kt
package com.vortexdbg.file

import java.io.File
import java.io.FileFilter

class VortexdbgFileFilter : FileFilter {

    override fun accept(pathname: File): Boolean {
        return !pathname.name.startsWith(UNIDBG_PREFIX)
    }

    companion object {
        const val UNIDBG_PREFIX = "__ignore.vortexdbg"
    }
}
